ludwig/explain/captum.py

```diff
@@ -142,18 +144,39 @@ def explain(self) -> Tuple[List[Explanation], List[float]]:
         # Compute attribution for each possible output feature label separately.
         expected_values = []
         for target_idx in tqdm(range(self.vocab_size), desc="Explain"):
-            total_attribution, feat_to_token_attributions = get_total_attribution_with_retry(
+            total_attribution, feat_to_token_attributions, total_attribution_global = get_total_attribution_with_retry(
                 self.model,
                 self.target_feature_name,
                 target_idx if self.is_category_target else None,
                 inputs_encoded,
                 baseline,
-                self.use_global,
                 len(self.inputs_df),
```

Comment:
Ok looks like we may not be using a sampled df - though I vaguely remember somewhere else in the explanation code we sample it.